payments/api.py

```diff
@@ -104,7 +104,6 @@ def generate_checkout_payload(request):
 def fulfill_completed_order(order, payment_data, basket=None):
     """Fulfill the order."""
     order.fulfill(payment_data)
-    order.save()
```

Comment:
I removed a bunch of these save calls since save is called in the order-model's methods.